res/values-ru/translatable.xml

```diff
@@ -57,35 +57,33 @@
   <string name="open">Открыть</string>
   <string name="go_home">Начальная папка</string>
   <string name="jumping_to_song">Дорожка уже в очереди, следующий…</string>
+  <!--See http://unicode.org/reports/tr35/tr35-numbers.html#Language_Plural_Rules and
```

Comment:
We are using Transifex for all translations, as described here: https://github.com/vanilla-music/vanilla#translating

Could you please revert this and add your translations in transifex? They will get included in the build the next time i trigger a sync.

Reply:
Will do.

Reply:
I requested joining the Transifex project.